CardScan/Classes/OcrDD.swift

```diff
@@ -9,6 +9,7 @@ import Foundation
 
 
 public class OcrDD{
+    public var lastDetectedBoxes: [CGRect] = []
     public init(){}
```

Comment:
do we need this init if it's not used?

Reply:
I need to add some code here as in Ocr.swift. Thanks for pointing this out.